requirements.txt

```diff
@@ -1,5 +1,6 @@
 Flask==2.2.2
 Flask-Login==0.6.2
+python-cas==1.6.0
```

Comment:
The Python CAS library is required for integration. Pick it up with `pip install -r requirements.txt`.